e2e/test/scenarios/collections/permissions.cy.spec.js

```diff
@@ -353,6 +353,29 @@ describe("collection permissions", () => {
               cy.signIn(user);
             });
 
+            it("should not show write access options on an empty, read-only collection", () => {
+              cy.request("GET", "/api/collection").then(xhr => {
+                // We need to obtain the ID programatically
+                const { id: THIRD_COLLECTION_ID } = xhr.body.find(
+                  collection => collection.slug === "third_collection",
+                );
+
+                // read-only, empty collection
+                cy.visit(`/collection/${THIRD_COLLECTION_ID}`);
+                cy.findByTestId("collection-empty-state")
+                  .findByText("Create a new…")
+                  .should("not.exist");
+              });
+
+              // writable, empty collection
+              cy.findByTestId("main-navbar-root")
+                .findByText("Your personal collection")
+                .click();
+              cy.findByTestId("collection-empty-state")
+                .findByText("Create a new…")
+                .should("exist");
+            });
```

Comment:
I think this is adequately unit tested, making this e2e test redundant. What do you think @nemanjaglumac ?

Reply:
100%.
We can/should remove E2E tests in this case.